foreman/data_refinery_foreman/foreman/management/commands/rerun_salmon_old_samples.py

```diff
@@ -0,0 +1,108 @@
+""" This management command will search for experiments where samples haven't been processed and
+that multiple versions of salmon have been used on the samples.
+"""
+
+import random
+import sys
+import time
```

Comment:
`random` and `time` aren't used by this command. Are any other of these imports unnecessary?